cpp/box/Box.h

```diff
@@ -476,15 +476,12 @@ class Box
     void contains(const vec3<float>* points, const unsigned int n_points, bool* contains_mask) const
     {
         util::forLoopWrapper(0, n_points, [&](size_t begin, size_t end) {
-            for (size_t i = begin; i < n_points; ++i)
-            {
-                std::transform(&points[begin], &points[end], &contains_mask[begin],
-                               [this](const vec3<float>& point) -> bool {
-                                   vec3<int> image(0, 0, 0);
-                                   getImage(point, image);
-                                   return image == vec3<int>(0, 0, 0);
-                               });
-            }
+            std::transform(&points[begin], &points[end], &contains_mask[begin],
```

Comment:
@klywang This is partly my fault as a reviewer on your previous work on this feature. The `std::transform` is meant to replace the raw `for` loop, not be nested inside of it. Also there was an issue that the `for` loop ran from `begin` to `n_points` instead of `begin` to `end`.

Reply:
Ooops :)